lib/bn.js

```diff
@@ -450,6 +450,11 @@ BN.prototype.toString = function toString(base, padding) {
   }
 };
 
+BN.prototype.toNumber = function toNumber() {
+  assert(this.bitLength() <= 53, 'Number can only safely store up to 53 bits');
+  return parseInt(this.toString());
```

Comment:
toString() defaults to radix 10 and padding 1. In what case would it result in leading zeroes (apart from the number 0)?